_acs-aem-commons/features/packagers/asset-packager/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: acs-aem-commons_feature
+title: Asset Packager
+description: Easily package pages and the associated assets
+redirect_from: /acs-aem-commons/features/asset-packager.html
+date: 2019-05-20
+feature-tags: administration
+initial-release: 4.2.0
+---
+
+# Purpose
+
+The Asset Packager lets you create a content package that is given a specific page path and will iterate over the content of that package to add all referenced assets into the package as well, giving you exactly what you need for a single page with all images.
+
+Helps a developer looking to:
+* Replicate an issue specific to content on a remote server
+* Not bloat your local instance unnecessarily
+* Still get enough content to replicate the issue (and not have broken images)
+
+or just a developer looking to get set up with some local content and want some but _not all_ of the DAM.
+
+
+# How to Use
+
+* Log in to AEM Author
+* Navigate to the Classic UI Tools Console (from the Touch UI, this is Tools:Operations:Configuration)
+* Under the `acs-commmons` folder, create a folder named `packagers` (NOTE: As of 1.6.0, this folder is created automatically)
```

Comment:
No need for this bullet point since this feature doesnt exist back in 1.6.0 (which is very distant past anyway) :)

Reply:
Updated in the latest commits